### PR TITLE
 Standardized Metabase db_service_name across dashboards

### PR DESCRIPTION
**Pull request by ayush-shah**
### Describe your changes :

`db_service_name` should be standardized around Dashboards in order to create lineage.

#
### Type of change :

- [x] Bug fix
- [x] Improvement

#
### Frontend Preview (Screenshots) :
<p align="center">For frontend related change, please link screenshots of your changes preview! Optional for backend related changes.
</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [ ] I have performed a self-review of my own. 
- [ ] I have tagged my reviewers below.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers

 @harshach @pmbrull

### CODE REVIEW

File: ingestion/src/metadata/ingestion/source/metabase.py [hunk 1]
```diff
@@ -57,7 +57,7 @@ class MetabaseSourceConfig(ConfigModel):
     chart_pattern: IncludeFilterPattern = IncludeFilterPattern.allow_all()
     service_name: str
     service_type: str = DashboardServiceType.Metabase.value
-    database_service_name: str = None
+    db_service_name: str = None
```
Comment (pmbrull) [inline ingestion/src/metadata/ingestion/source/metabase.py:60]: Let's wrap this as `Optional[str]` to get rid of the smell

Comment (harshach) [reply ingestion/src/metadata/ingestion/source/metabase.py:60]: @ayush-shah can you take care of this.

Comment (ayush-shah) [reply ingestion/src/metadata/ingestion/source/metabase.py:60]: yes, on it.

**Comment by ayush-shah** [comment]
Resolved all the comments. 

cc: @pmbrull @harshach

**Review by pmbrull — APPROVED** [review]
LGTM. Note that you could keep the assignment of None after wrapping db service name in an Option

**Pull request merged**